MANUAL_INSTALLATION.md

```diff
@@ -15,62 +11,55 @@ can be verified (see below).
 
 ## Installation overview
 
-We recommend you extract the package to `~/.cdk`, and use the binaries from `~/.cdk/bin`.
+After extracting the archive, follow the following instructions:
 
-When using the manual installation method, use the command `y-npm` (provided
-with the distribution) wherever you would normally use `npm`.
+### Installing the CLI
 
-## Step by step instructions
+The CLI (the `cdk` command) is always installed using npm, regardless of your language.
 
-### Extract the installation archive to ~/.cdk
+```
+npm install -g /path/to/zip/js/aws-cdk-1.2.3.tgz
```

Comment:
This is going to resolve all dependencies of it from internet... Including @aws-cdk/cx-api, foe example. I don't think we fit the bill of a "manual" install from the released packages anymore...